fraud/statestreet-legacy-schema.groovy

```diff
@@ -1,5 +1,5 @@
 // set RF=1 due to limited space
-system.graph('statestreet').replication("{'class' : 'NetworkTopologyStrategy', 'SearchGraphAnalytics' : 1}").systemReplication("{'class' : 'NetworkTopologyStrategy', 'SearchGraphAnalytics' : 1}").create()
+system.graph('statestreet').replication("{'class' : 'NetworkTopologyStrategy', 'SearchGraphAnalytics' : 1}").systemReplication("{'class' : 'NetworkTopologyStrategy', 'SearchGraphAnalytics' : 1}").engine(Classic).create()
```

Comment:
is it possible to catch the error here and retry without `engine(Classic)` to make it work with 6.0 and 6.7?

Reply:
do you already have specific code to do that? Otherwise I'll have to look into that on monday

Reply:
monday is okay, probably no tests will work over the weekend anyway :)

Reply:
https://datastax.jira.com/browse/DSP-18829 will be the solution